packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -134,8 +134,18 @@ export class DefaultEventHandlers<O = {}> extends CoreEventHandlers<
           }
         );
 
+        const unbindMouseleave = this.addCraftEventListener(
+          el,
+          'mouseleave',
+          (e) => {
+            e.craft.stopPropagation();
+            store.actions.setNodeEvent('hovered', '');
```

Comment:
Use `null` instead:

```suggestion
            store.actions.setNodeEvent('hovered', null);
```

Reply:
Yeah, Craft is using Typescript without strict mode thus this weird inconsistency. I will create a separate PR to migrate the repo to strict mode, but for now I've updated your PR to be consistent with the rest of the codebase.